BookPlayer/Library/ItemListViewController.swift

```diff
@@ -138,9 +143,24 @@ class ItemListViewController: UIViewController, ItemList, ItemListAlerts, ItemLi
         self.animateView(self.bulkControls, show: editing)
         self.tableView.setEditing(editing, animated: true)
 
+        if editing {
+            previousLeftButtons = navigationItem.leftBarButtonItems
+            navigationItem.leftBarButtonItems = [selectButton]
+            selectButton.isEnabled = tableView.numberOfRows(inSection: 0) > 0
+            updateSelectButtonTitle()
+        } else {
+            navigationItem.leftBarButtonItems = previousLeftButtons
+        }
+
         NotificationCenter.default.post(name: notification, object: nil, userInfo: nil)
     }
 
+    func updateSelectButtonTitle() {
+        self.selectButton.title = self.tableView.numberOfRows(inSection: 0) > (self.tableView.indexPathsForSelectedRows?.count ?? 0)
+            ? "Select All"
+            : "Deselect All"
```

Comment:
following the line from my previous comment, we would only need to add this to the function

```swift
      guard self.tableView.indexPathForSelectedRow == nil else {
            self.bulkControls.moveButton.isEnabled = true
            self.bulkControls.trashButton.isEnabled = true
            return
        }

        self.bulkControls.moveButton.isEnabled = false
        self.bulkControls.trashButton.isEnabled = false
```